src/backend/commands/explain.c

```diff
@@ -1317,7 +1281,7 @@ ExplainNode(PlanState *planstate, List *ancestors,
 			break;
 		case T_NestLoop:
 			pname = sname = "Nested Loop";
-			if (((NestLoop *)plan)->shared_outer)
+			if (((NestLoop *) plan)->shared_outer)
```

Comment:
Please remove such meaningless changes. Save pgindent formatting only for your code blocks.

Reply:
Fixed